components/generic/emails-input/index.tsx

```diff
@@ -0,0 +1,160 @@
+import * as EmailValidator from 'email-validator'
+
+import { memo, useEffect, useState } from 'react'
+
+function Email(props) {
+  const valid = EmailValidator.validate(props.email)
+  return (
+    <>
+      <div className={`email ${!valid ? 'error' : ''}`}>
+        {props.email}{' '}
+        <span className="close" onClick={props.onClickRemove}>
+          <svg
```

Comment:
it's better not to use svg code directly in the component. it's better to create .svg file and import it whenever it needs